typescript/src/ethereum.ts

```diff
@@ -762,6 +763,42 @@ export class Bridge
       )
     )
   }
+
+  // eslint-disable-next-line valid-jsdoc
+  /**
+   * @see {ChainBridge#getDepositRevealedEvents}
+   */
+  async getRedemptionRequestedEvents(
+    options?: GetEvents.Options,
+    ...filterArgs: Array<unknown>
+  ): Promise<RedemptionRequestedEvent[]> {
+    const events: EthersEvent[] = await this.getEvents(
+      "RedemptionRequested",
+      options,
+      ...filterArgs
+    )
```

Comment:
It probably won't work if we want to filter events by the indexed `walletPubKeyHash` param. See https://github.com/ethers-io/ethers.js/pull/4244 - I ran into this issue in T daap with ethers `v5.4.7`. Here we use `v5.5.2` but I think this problem still exists in this version. As a workaround, you can take a look at [this](https://github.com/threshold-network/token-dashboard/blob/main/src/threshold-ts/tbtc/index.ts#L1041C1-L1093C1).

Reply:
Added a note about that: cce9d9c6. Thanks!